apps/heft/src/pluginFramework/InternalHeftSession.ts

```diff
@@ -13,11 +13,13 @@ import type { LoggingManager } from './logging/LoggingManager';
 import type { HeftConfiguration } from '../configuration/HeftConfiguration';
 import type { HeftTask } from './HeftTask';
 import type { HeftParameterManager } from './HeftParameterManager';
+import { IHeftParsedCommandLine } from './HeftTaskSession';
```

Comment:
```suggestion
import type { IHeftParsedCommandLine } from './HeftTaskSession';
```

Reply:
Is this meaningful? An interface is an inherently type-only